config/settings/production.py

```diff
@@ -19,6 +19,7 @@
         dsn=SENTRY_DSN, integrations=[DjangoIntegration()], send_default_pii=True
     )
 
+GA_TAG_KEY = env.str("GOOGLE_GA_TAG_KEY", "REPLACE_ME")
```

Comment:
This is the same in `base.py`, so I don't think we need it here as well.

If we want to break if this isn't present (eg. we don't just not load the GA tag if this config value isn't present), then we should replace this with `env.str("GOOGLE_GA_TAG_KEY")`, so that the app won't start without this config value.